googleapiclient/model.py

```diff
@@ -273,13 +273,19 @@ def serialize(self, body_value):
         return json.dumps(body_value)
 
     def deserialize(self, content):
+        body = {}
```

Comment:
Shouldn't be needed (see below).

Reply:
That was to cope with "Local variable 'body' might be referenced before assignment" warning of my IDE, but I agree with removing it.